tests/e2e/sample-applications/mongo-persistent/mongo-persistent-block.yaml

```diff
@@ -129,6 +129,25 @@ items:
               volumeDevices:
                 - name:  block-volume-pv
                   devicePath: /dev/xvdx
+              livenessProbe:
+                tcpSocket:
+                  port: mongo
+                initialDelaySeconds: 5
+                periodSeconds: 10
+              startupProbe:
+                exec:
+                  command:
+                  - mongosh
+                  - admin
+                  - -u $(MONGO_INITDB_ROOT_USERNAME)
+                  - -p $(MONGO_INITDB_ROOT_PASSWORD)
+                  - --eval 'printjson(db.getCollectionNames())'
+                initialDelaySeconds: 5
+                periodSeconds: 30
+                timeoutSeconds: 2
+                successThreshold: 1
+                failureThreshold: 40 # 40x30sec before restart pod
```

Comment:
this big failure threshold is to avoid it to fail? just not setting it wouldn't do the trick?

Reply:
So we want the timeout set on Gingko for example [here](https://github.com/openshift/oadp-operator/blob/master/tests/e2e/backup_restore_suite_test.go#L136)  to fail before the pod timeout is up, so gingko reports it as failed. 20min is in the situation where we want to really wait 19min on the gingko side as but I don't have clear number of how much wait we should really wait.